packages/cf-component-textarea/src/TextareaTheme.js

```diff
@@ -0,0 +1,32 @@
+export default baseTheme => ({
+  height: 'initial',
+  wordBreak: 'initial',
+  cursor: 'initial',
+  resize: 'initial',
+
+  zIndex: 0,
+  width: '100%',
+  margin: '0 0 0.75rem',
+  padding: '0.45em 0.75em',
+  border: '1px solid #bdbdbd',
```

Comment:
I'm mostly just copy pasting this code over from our legacy CSS so I'm not really checking the values here but going over this I can see it's indistinguishable from they 'hail' color.